### ISSUE TITLE
Broken link in buildserver documentation

### ISSUE THREAD

**Issue by Chemin1** [ElektraInitiative/libelektra#2893]
A link in BUILDSERVER.md is broken. This leads to [failures](https://github.com/ElektraInitiative/libelektra/pull/2626/checks?check_run_id=196312326) in the continuous integration.

```
Broken Links:
—————————————
doc/BUILDSERVER.md:294:0| https://build.libelektra.org/job/elektra-homepage/
—————————————
```

According to the [list](https://build.libelektra.org/jenkins/) it should be `elektra-website` instead of `elektra-homepage`. I don't know why this does not break other builds, as the link really does not work.

**Comment by markus2330** [comment]
Thank you for reporting this problem!

I actually renamed the build server job to elektra-website (because the tool is also called website now). Unfortunately, #2885 (which fixes the link) was not ready to merge because of other problems.

But I think @sanssecours already pushed a commit to fix this problem to master e00fb6cd2ec4ee062976bbc4637a804e0be4b3ac so this issue should be resolved now.

**Comment by Chemin1** [comment]
> But I think @sanssecours already pushed a commit to fix this problem to master [e00fb6c](https://github.com/ElektraInitiative/libelektra/commit/e00fb6cd2ec4ee062976bbc4637a804e0be4b3ac) so this issue should be resolved now.

He did, my last fetch from upstream was some hours too old. Sorry for the unnecessary issue.

**Comment by markus2330** [comment]
No, really thank you for reporting. It is always good to have discussions about these issues. If we always simply silently fix it, we might miss opportunities to improve our workflow.
